content/en/synthetics/api_tests/icmp_tests.md

```diff
@@ -0,0 +1,157 @@
+---
+title: ICMP Tests
+kind: documentation
+description: Monitor availability of your hosts and diagnose network issues
+aliases:
+  - /synthetics/icmp_test
+  - /synthetics/icmp_check
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetic Monitoring"
+- link: "/getting_started/synthetics/api_test"
+  tag: "Documentation"
+  text: "Get started with API tests"
+- link: "/synthetics/private_locations"
+  tag: "Documentation"
+  text: "Run ICMP Pings on internal endpoints"
+---
+
+
+## Overview
+
+ICMP tests allow you to easily **monitor the availability of your hosts and diagnose network communication issues to verify responsiveness from your application's endpoints**. By asserting on the values received from one or more **ICMP ping(s)** to your endpoint, Datadog can help **detect availability/connectivity issues, above-quota latency or hops, and changes in security firewall configuration**. ICMP tests can run from [managed][1] and [private locations][2] depending on whether you are willing to trigger ICMP pings to your endpoints from outside or inside your network.
```

Comment:
```suggestion
ICMP tests allow you to easily **monitor the availability of your hosts** and **diagnose network communication issues**. By asserting on the values received from one or more **ICMP ping(s)** to your endpoint, Datadog can help **detect connectivity issues, above-quota latency or hops, and unexpected changes in security firewall configuration**.  
 
ICMP tests can run from [**managed**][1] and [**private locations**][2] depending on whether you are willing to trigger ICMP pings to your endpoints from outside or inside your network.
```

Can you also add the bit about the type of runs? (ie. scheduled, on demand, etc.).